test.rb

```diff
@@ -0,0 +1,45 @@
+require 'json'
+require_relative 'person'
+# file = File.open("test.json")
+# file_data = File.read("test.txt").split(',')
+
+# puts file_data[0]
+
+# arr = [1, 2, 3]
+# json = JSON.generate(arr)
+#
+
+person = Person.new(age: 10, name: 'amine')
+# person = {
+#     "name":
+#     "age":
+#     "permission":
+#     "specialization":
+#     "rentals":
+
+# }
```

Comment:
:bulb:

Reply:
😮 I didn't realize that this file was pushed. I thought it was removed after writing the `git stash` command. I will remove it.